draft-ietf-sidrops-aspa-notation-02.md

```diff
@@ -84,14 +84,27 @@ to use their own notation styles instead of, or in addition to this.
 This specification uses ABNF syntax specified in [@!RFC5234].
 
 ~~~
-notation           = customer-asid separator providers
-customer-asid      = asn
-separator          = " => "
-providers          = provider-as *(provider-separator provider-as)
-provider-as        = asn
-provider-separator = ", "
-asn                = ["AS"] uint32
-uint32             = %d0-4294967295
+notation            = customer-asid separator providers
+
+customer-asid       = asn
+separator           = " => "
+
+providers           = providers-one-line / providers-multiline
+providers-one-line  = asn *(*wsp "," *wsp asn)
+providers-multiline = "[" *wspml asn *(*wspml "," *wspml asn) *wsp "]"
```

Comment:
Does `wsp` allow the final newline in examples like 
```
AS65000 => [
    AS65001,
    AS65002,
    AS65003
]
```?

Reply:
Indeed - this should be wspml (also mentioned in the list somewhere or in direct email to me).

But, I will merge this first and then make a new branch for -03 and fix it there.